ios/FluentUI/PreviewCard/Attachment.swift

```diff
@@ -0,0 +1,133 @@
+        hostingController = FluentUIHostingController(rootView: AnyView(attachmentView.windowProvider(self)))
+        hostingController.disableSafeAreaInsets()
+        hostingController.view.backgroundColor = .clear
+    }
+    var window: UIWindow? {
+        return self.view.window
+    }
+    private var hostingController: FluentUIHostingController!
+    private var attachmentView: Attachment!
+}
+
+/// View that represents the Attachment.
+public struct Attachment: View {
+    var isElevated: Bool
+    var thumbnail: UIImage
+    @Environment(\.theme) var theme: FluentUIStyle
+    @Environment(\.windowProvider) var windowProvider: FluentUIWindowProvider?
+    /// Creates Attachment's thumbnail view.
+    @ViewBuilder
+    var attachmentThumbnail: some View {
+        HStack {
+            Image(uiImage: thumbnail)
+                .frame(width: Constants.attachmentThumbnailWidth, height: Constants.attachmentThumbnailHeight)
+                .padding(EdgeInsets(top: Constants.attachmentThumbnailPaddingTop, leading: Constants.attachmentThumbnailPaddingLeading, bottom: Constants.atachmentZeroValue, trailing: Constants.atachmentZeroValue))
+        }
+    }
+    /// Creates Attachment's entire text view.
+    @ViewBuilder
+    var attachmentText: some View {
+        VStack(alignment: .leading) {
+            Text("Text")
+                .font(.body)
+                .foregroundColor(Color(red: Constants.attachmentTextColor, green: Constants.attachmentTextColor, blue: Constants.attachmentTextColor))
+                .frame(width: Constants.attachmentTextWidth, height: Constants.attachmentTextHeight, alignment: .leading)
+            Text("Subtext")
+                .font(.caption)
+                .foregroundColor(Color(red: Constants.attachmentSubtextColor, green: Constants.attachmentSubtextColor, blue: Constants.attachmentSubtextColor))
+                .frame(width: Constants.attachmentSubtextWidth, height: Constants.attachmentSubtextHeight, alignment: .leading)
+                .padding(EdgeInsets(top: Constants.attachmentSubtextPaddingTop, leading: Constants.atachmentZeroValue, bottom: Constants.atachmentZeroValue, trailing: Constants.atachmentZeroValue))
+        }
+        .frame(width: Constants.attachmentEntireTextWidth, height: Constants.attachmentEntireTextHeight)
+        .padding(EdgeInsets(top: Constants.atachmentZeroValue, leading: Constants.attachmentEntireTextPaddingLeading, bottom: Constants.atachmentZeroValue, trailing: Constants.atachmentZeroValue))
+    }
+    /// Creates inner Attachment view containing the thumbnail and text.
+    @ViewBuilder
+    var innerContents: some View {
+        HStack(spacing: 0) {
+        attachmentThumbnail
+        attachmentText
+        Spacer()
+        }
+        .frame(minWidth: Constants.attachmentWidth, minHeight: Constants.attachmentHeight)
+    }
+    public var body: some View {
+        innerContents
+                /// Creates a background that matches the PreviewCard.
```

Comment:
This is fine for now, but once we start to combine these controls we should probably consolidate this to only have one background, not two.

Reply:
No problem, I have code ready that will resolve this.